front-end/src/app/reports/f3x/create-workflow/create-f3x-step1.component.ts

```diff
@@ -173,14 +176,22 @@ export class CreateF3XStep1Component extends DestroyerComponent implements OnIni
     this.router.navigateByUrl('/reports');
   }
 
-  public save(jump: 'continue' | undefined = undefined) {
+  public async save(jump: 'continue' | undefined = undefined) {
     this.formSubmitted = true;
     if (this.form.invalid) {
       this.store.dispatch(singleClickEnableAction());
       return;
     }
 
     const summary: Form3X = Form3X.fromJSON(SchemaUtils.getFormValues(this.form, f3xSchema, this.formProperties));
+    if (this.committeeAccount) {
+      summary.committee_name = this.committeeAccount.name;
```

Comment:
I like the idea of assigning the committee_name here.  The requirements are to assign it on the session where the report is submitted for submission/webprint/test-.fec so that it reflects any changes made up to that point.  We handled this for `qualified_committee` by running an update before we kick off any .fec tasks:

```
async generate() {
    if (this.report instanceof Form3X) {
      const payload: Form3X = Form3X.fromJSON({
        ...this.report,
        qualified_committee: this.form3XService.isQualifiedCommittee(this.committeeAccount)
      });
      await firstValueFrom(this.form3XService.update(payload, ['qualified_committee']));
    }
    this.apiService.post(`/web-services/dot-fec/`, { report_id: this.report?.id }).subscribe(() => undefined);
    this.fileIsGenerated = true;
  }
```

You could tie in committee_name there too (especially since `this.committee_account` will be available in the scope.